source/libsodium/package.d

```diff
@@ -1,30 +1,27 @@
-/*******************************************************************************
-
-    D language bindings for libsodium's sodium.h
-
-    License: ISC (see LICENSE.txt)
-
-*******************************************************************************/
-
+/// D language bindings for libsodium
+///
+/// License: ISC (see LICENSE.txt)
 module libsodium;
 
+public import libsodium;
```

Comment:
Why does this import itself ?

Reply:
Oops, accidentally removed ignorance of it

Reply:
Done in 66944af